pkg/machinepool/helper.go

```diff
@@ -390,3 +405,54 @@ func (r *ReplicaSizeValidation) MaxReplicaValidatorOnClusterCreate() interactive
 		)
 	}
 }
+
+func GetZoneType(machinePool *cmv1.MachinePool) string {
+	zones := machinePool.AvailabilityZones()
+	if len(zones) == 0 {
+		return ZoneTypeNA
+	}
+
+	for _, zone := range zones {
+		zoneLower := strings.ToLower(zone)
+
+		if strings.Contains(zoneLower, "wl") {
+			return ZoneTypeWavelength
+		}
+		if strings.Contains(zoneLower, "outpost") {
+			return ZoneTypeOutpost
+		}
+		if strings.Contains(zoneLower, "-lz") || strings.Count(zoneLower, "-") > 3 {
+			return ZoneTypeLocalZone
+		}
+	}
+
+	return ZoneTypeStandard
+}
+
+func IsWinLIEnabled(labels map[string]string) string {
+	if val, ok := labels[LabelImageType]; ok && strings.ToLower(val) == ImageTypeWindows {
+		return BoolStringTrue
+	}
+	return BoolStringFalse
+}
+
+func IsDedicatedHost(machinePool *cmv1.MachinePool, r *rosa.Runtime) string {
+	aws := r.AWSClient
+	awsMachinePoolID := machinePool.AWS().ID()
+	if awsMachinePoolID == "" {
+		return BoolStringFalse
+	}
+
+	hasDedicatedHost, err := aws.CheckIfMachinePoolHasDedicatedHost([]string{awsMachinePoolID})
+	if err != nil {
+		if r.Reporter != nil {
+			_ = r.Reporter.Errorf("Failed to check dedicated host status: %v", err)
+		}
```

Comment:
The reporter is expected to be not-nil in every command, is there a reason it could be nil here? (Is this function called somewhere where the reporter is not passed in?) I think we can remove this check since it's implicit it will exist 🙂